src/Robo/Plugin/Commands/DevelopmentModeBaseCommands.php

```diff
@@ -123,43 +123,61 @@ public function databaseRefreshLando(string $siteName = 'default'): Result
     /**
      * Refresh database on Tugboat.
      */
-    public function databaseRefreshTugboat(): Result
+    public function databaseRefreshTugboat(): ResultData
     {
         $this->io()->title('refresh tugboat databases.');
-        $result = null;
+        $result_data = new ResultData();
+
         foreach (array_keys($this->getAllSitesConfig()) as $siteName) {
             try {
                 $dbPath = $this->databaseDownload($siteName);
-            } catch (TaskException) {
+            } catch (TaskException $te) {
                 $this->yell("$siteName: No database configured. Download/import skipped.");
+                $result_data->append($te->getMessage());
```

Comment:
```suggestion
                $resultData->append($te->getMessage());
```

[Symfony coding standards want camel case for variable names](https://symfony.com/doc/current/contributing/code/standards.html#naming-conventions).

Reply:
Ok. I'll apply this throughout the function. Thank you.